lib/pluginTapestry.js

```diff
@@ -186,8 +186,6 @@ const register = function (server, pluginOptions) {
   // where to log messages
   var config = server.methods.getService('tapestry')
   config.cache = server.methods.getConfig().cache
-  // We've seen requests logging ~500Kb to loggly with the Tapestry logging turned on, this when multiplied by the 22k packageRates requests, is filling up loggly and providing no value. Please carefully consider before enabling the logger.
-  config.logger = server.methods.getDataLoggingWrapper('tapestry')
```

Comment:
didn't @jask84 add this in for his remove HXCM for tapestry move work?

Reply:
Looking at the comments we did turn it on for that project, it's not currently providing value though.

Reply:
ok 👍

Reply:
spoke with Kev about this, we send more useful logs from the-works now anyway - https://github.com/holidayextras/the-works/blob/staging/flows/tapestryService.js#L77